index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="css-reset.css" />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <!-- Navigation bar -->
+    <nav id="nav-bar">
+      <p class="top-text">Already have a an intuit Account?</p>
+      <a class="bottom-text" href="#">I forgot my user ID or password</a>
+
+      <button class="sign-button" type="button">
+        <img class="lock-image" src="img/lock.png" alt="lock" />
+        Sign in
+      </button>
+    </nav>
+
+    <!-- form section -->
+
+    <section id="form">
+      <img class="logo" src="img/logo-intuit.png" alt="img" />
+      <ul class="logos">
+        <li class="mqt">
+          <img class="lower-imgs" src="img/mint.png" alt="img" />
+          <p>mint</p>
+        </li>
+        <li class="mqt">
+          <img class="lower-imgs" src="img/qb_thumb.png" alt="img" />
+          <p>quickbooks</p>
+        </li>
+
+        <li class="mqt">
+          <img class="lower-imgs" src="img/turbotax.png" alt="img" />
+          <p>turbotax</p>
+        </li>
+      </ul>
+      <form action="get">
+        <h3 class="form-header">Create an Intuit account</h3>
+        <p>One account for everything Intuit, including Mint.</p>
+        <p class="learn-more"><a href="">Learn more</a></p>
+        <div class="form-element">
+          <label for="email">Email</label>
+          <input id="email" type="email" />
+        </div>
+        <div class="form-element">
+          <label for="phone">Phone (recommended)</label>
+          <input id="phone" type="tel" />
+        </div>
+        <div class="form-element">
+          <label for="password">Password</label>
+          <input id="password" type="password" />
+        </div>
+
+        <!-- buttons -->
+
+        <button id="create-account" type="submit">
+          <img class="lock-image" src="img/lock.png" alt="lock" />Create Account
+        </button>
+        <p>
+          By clicking Create account, you agree to our <a href="">Terms</a> and
+          have read a <a href="">Privacy </a>
+          <br />
+          <a href="#">Statement</a> .
+        </p>
+        <footer class="form-footer">
+          <p>
+            <span class="recaptcha"> Invisible reCAPTCHA by Google</span>
+            <a href="#">Privacy Policy </a><span class="recaptcha"> and</span>
+            <a href="#">Terms of Use</a> .
+          </p>
+        </footer>
```

Comment:
- [ ] **[OPTIONAL]** The reason why the dark gray part doesn't take the complete space, is because of the padding of your `<form>` container, you can solve it by taking your footer out of the form and applying the corresponding rules to the footer
![image](https://user-images.githubusercontent.com/53256139/87428898-9ef11a00-c5a8-11ea-8e52-7f64d7972b37.png)